packages/fast-components-styles-msft/src/button/index.ts

```diff
@@ -49,13 +58,16 @@ const applyTransparentBackplateStyles: CSSRules<DesignSystem> = {
         "& $button_contentRegion::before": {
             background: neutralForegroundRest,
             height: toPx<DesignSystem>(focusOutlineWidth),
+            [highContrastSelector]: {
+                background: "ButtonText",

```

Comment:
Looks like a couple of places that should use the constants.

Reply:
thanks, added